src/components/Clients/ClientListTable.jsx

```diff
@@ -7,26 +7,28 @@ import TableContainer from '@mui/material/TableContainer';
 import TableHead from '@mui/material/TableHead';
 import TableRow from '@mui/material/TableRow';
 import Paper from '@mui/material/Paper';
-// Custom Hook Imports
-import { useStatusNotification } from '../../hooks';
 // Context Imports
 import { UserListContext } from '../../contexts';
 // Component Imports
 import ClientListTableRow from './ClientListTableRow';
 import { StyledTableCell } from '../Table/TableStyles';
 
 // ===== MAKE CHANGES HERE FOR TABLE HEADER / COLUMN TITLES =====
-const columnTitlesArray = ['Select', 'Client', 'WebID', 'Pin', 'Delete'];
+const columnTitlesArray = ['Client', 'Client Profile', 'Pin', 'Delete'];
```

Comment:
This Client Profile link is a great change. It removes the Web ID from the view while still keeping its utility

![image](https://github.com/codeforpdx/PASS/assets/37914436/fbf9353f-8896-4991-b220-6b529bb30726)